pages/index.js

```diff
@@ -12,6 +16,29 @@ import PromoSection from '../src/PromoSection'
 import UseCases from '../src/UseCases'
 import Subscribe from '../src/Subscribe'
 
+/**
+ * HomePage fn component
+ */
+const HomePage = () => (
+  <Page stickHeader={true}>
+    <HeadInjector />
+    <Hero>
+      <LandingHero />
+      <LearnMoreSection>
+        <LearnMore />
+      </LearnMoreSection>
+      <span name="nextSlide" style={{ marginTop: `-58px` }} />
+    </Hero>
+    <Diagram />
+    <PromoSection />
+    <UseCases />
+    <Subscribe />
+  </Page>
+)
+
+/**
+ * HeadInjector fn component
+ */
 const HeadInjector = () => (
```

Comment:
again, not clear why this change - I mean switching the order

Reply:
Same as https://github.com/iterative/dvc.org/pull/776#discussion_r342929399. Just looking for a consistent organization of code in component files. Sorry, should've done this in a separate PR though... It may be a little difficult to un-do just these changes at this point, but I can manage to, if you prefer. Please lmk.

Reply:
no, not a problem see my comment above. Just check if there is an actual style guide exists around organizing exports. It might that both ways are just fine.

Reply:
This was actually not an export, just a minor (kind of unnecessary) private component, so I moved it under the main export in the file.

Reply:
another thing to keep in mind is that it complicates reading the code a bit, you don't expect something being used defined after the usage - it's not natural

Reply:
Yes but we define all the styled components under the main exports. Should we also place those above? Then you'd have to scroll a lot to get to the export, also complicating the reading. Not sure which option is best or whether arbitrarily putting some on top and some later was actually the best choice...

I personally like having the export all the way on top for readability, it probably be the best entry point to reading a file in most cases anyway.

Reply:
> Should we also place those above?

Def not. `styled` - they are not so much about code even, it is a very specific thing and there are specific conventions around it. I would not use them as an example. Usually styles are located in a separate file.

> I personally like having the export all the way on top for readability, it probably be the best entry point to reading a file in most cases anyway.

it makes sense. I'm just sharing that for me it would a bit counterintuitive to expect definitions after the code. I don't have a very strong opinion though and fine with either. @iAdramelk do you have any preferences or some JS wisdom to share? :)

Reply:
I agree that functions and methods should be defined before they are used; it helps readability. But I'm conflicted about styled-components. On the one hand, they will be in the way if we place them above the code as they didn't have any logic in them. On the other hand, I often feel confused, then I see something then looks like a react component, but I didn't see import or definition. Then I scroll, and I understand that this is just a styled component. Maybe some prefix to the styled component name could help there like starting their names with capital S or Styled.

Reply:
OK I reordered the pages/ files again to have local components defined before the export.

Reply:
> I often feel confused, then I see something then looks like a react component, but I didn't see import or definition.... Maybe some prefix to the styled component name could help there like starting their names with capital S or Styled.

Well for example, if you see `<Container>` in a component's render() method, and didn't see the definition, you can assume its defined later (else React will throw an error at runtime) – so it's probably a styled component. I don't oppose revising names though! I would recommend just we can just add the name of the styled HTML tag, in this case `<ContainerDiv>`. Should we create a separate issue for this?

Reply:
Extracted to #792.